### ISSUE TITLE
The sidebar did not collapse automatically after clicking on a SidebarItem.

### ISSUE THREAD

**Issue by codersjj** [sanyuan0704/island.js#186]
### Describe the bug

On narrow screen devices, the sidebar did not collapse automatically after clicking on a SidebarItem.

![sidebar-collapse](https://github.com/sanyuan0704/island.js/assets/44868357/b556fd48-8836-4d5c-ac59-0627f5e53f6d)

### Reproduction

See: https://codesandbox.io/p/sandbox/ancient-tdd-zlh9qn?file=%2Fisland-app%2Fdocs%2F.island%2Fconfig.ts%3A44%2C7

### Expected behavior

On narrow screen devices, the sidebar collapses automatically after clicking on a SidebarItem.

### System Info

```Shell
-
```

### Validations

- [X] Read the [docs](https://island.sanyuan0704.top)
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

**Comment by sanyuan0704** [comment]
Yes, we optimize this behavior by closing the sidebar after clicking other sidebar item. PR welcome!
